### ISSUE TITLE
Paste in Vim mode doesn’t work on Firefox

### ISSUE THREAD

**Issue by bpierre** [codesandbox/codesandbox-client#4878]
# 🐛 bug report

## Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

## Description of the problem

Pasting with the Vim mode using Ctrl+v triggers the following error:

> TaskQueue: Error running task. Failed to handle key=&lt;C-v>. navigator.clipboard.readText is not a function.

## How has this issue affected you? What are you trying to accomplish?

I would like to paste in Vim mode.

### To Reproduce

1. Use Firefox.
2. Activate Vim mode
3. Enter in edition mode (e.g. by pressing `i`).
4. Hit `Ctrl+v`.

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      | Latest
| Browser          | Firefox Nightly (latest)
| Operating System | Arch Linux

**Comment by SaraVieira** [comment]
Hey

I went really deep in this bug to try and figure out what was wrong since I tested in Firefox on my MacBook and it did work.

So I installed Virtual Box to test on Linux and indeed it does not work and it works in Chrome on Linux(ubuntu 1804), with WAAAAY more digging I found out that it's actually a Firefox on Linux issue that does not allow read from the clipboard in Javascript.

A friend of mine that works for electron helped me figure out that the way Firefox reads from the clipboard using something different to get the clipboard data, she even found the file:
 
https://github.com/mozilla/gecko-dev/blob/63ad13457c9b5e0757edded4999625f9466b8228/widget/gtk/nsClipboard.cpp

I filed an issue on Bugzilla firefox bugs tracker and let's hope they can fix this:
https://bugzilla.mozilla.org/show_bug.cgi?id=1664318


Long story short there is nothing we can do since this seems to be a firefox bug only on Linux :( 

Sorry about the issue, but I will close it

Thank you for this bug, I learned a lot!

**Comment by bpierre** [comment]
Ah too bad, I was [so close](https://github.com/codesandbox/codesandbox-client/issues/1704#issuecomment-687420019) to be able to use CodeSandbox in Vim mode again haha. Thanks for having spent the time to understand the issue and file a bug, I will follow it closely :eyes:

**Comment by SaraVieira** [comment]
I know, made me super sad too, it was so long we had that disabled and soooo close :(

If it does work I will mention it here too :)
